nox.py

```diff
@@ -116,8 +116,8 @@ def lint(session):
 
     session.run(
         'flake8',
-        '--exclude=opencensus/trace/exporters/gen/',
-        'opencensus/', 'tests/', 'examples/')
+        '--exclude=contrib/opencensus-ext-ocagent/opencensus/ext/ocagent/trace_exporter/gen/',
```

Comment:
Ideally we want to have a config file (e.g. lint.rc) under the ocagent ext folder. Flake doesn't seem to support this, probably we should revisit when we switch to pylint.